pkg/helm/helm.go

```diff
@@ -300,3 +309,16 @@ func (h *Hydrator) appendAuthArgs(ctx context.Context, args []string) ([]string,
 	}
 	return args, nil
 }
+
+// we determine if a version is a valid range by checking that (a) it is not
+// valid semver on its own and (b) that it can be parsed correctly as a version range
+func isRange(version string) bool {
+	if version == "" {
+		return true
+	}
+	if semver.IsValid("v" + version) {
+		return false
+	}
+	_, err := semverrange.NewConstraint(version)
```

Comment:
This is the same semver library that [helm uses](https://github.com/helm/helm/blob/4e447d87cd3222aa5e5ea2b38170e9d80b99c0b4/pkg/registry/util.go#L75) for range detection